torchsnapshot/storage_plugins/s3.py

```diff
@@ -7,13 +7,14 @@
 
 import io
 import os
+from typing import Any, Dict
 
 from torchsnapshot.io_types import ReadIO, StoragePlugin, WriteIO
 from torchsnapshot.memoryview_stream import MemoryviewStream
 
 
 class S3StoragePlugin(StoragePlugin):
-    def __init__(self, root: str) -> None:
+    def __init__(self, root: str, storage_options: Dict[str, Any]) -> None:
```

Comment:
The s3/gcs tests initialize storage plugins directly (the tests are skipped on fork pull requests). Let's make `storage_options` here optional?